manifest.toml

```diff
@@ -7,7 +7,7 @@ description.fr = "Plateforme de collaboration open source conçue pour les déve
 
 version = "9.4.2~ynh1"
 
-maintainers = ["pmorinerie"]
+maintainers = [ "pmorinerie" ]
```

Comment:
Could you remove unrelated changes from the toml, like the extra spaces and indentation changes ?

Reply:
I’m sorry it’s my automatic formatter

Reply:
done